engines/python/setup/djl_python/properties_manager/tnx_properties.py

```diff
@@ -51,6 +50,18 @@ class TnXModelLoaders(str, Enum):
     optimum = "optimum"
 
 
+class TnXModelSchema(str, Enum):
+    legacy = "legacy"
+    optimum = "optimum"
+    safetensors = "safetensors"
+    compile_only = "compile_only"
+
+
+class TnXGenerationStrategy(str, Enum):
+    continuous_batching = "continuous_batching"
+    naive_rolling_batch = "naive_rolling_batch"
```

Comment:
what's the difference between these two?

Reply:
Difference is that continuous batching uses the exposed KV cache implementation in transformers neuronx, and the naive implementation recalculates the prefill for all items in the batch every prefill step. It was the old workaround for not having KV cache available, but there are some models that do not support continuous batching so I brought it in as fallback support.